documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,301 @@
+
+..MTV-773
+
+..MTV-724
+
+..MTV-766
+
+..MTV-768
+
+..MTV-770
+
+..MTV-772
+
+..MTV-775
+
+..MTV-779
+
+..MTV-792
+
+..MTV-793
+
+..MTV-797
+
+..MTV-867
+
+.OVA Provider
+
+OVA provider. link:https://issues.redhat.com/browse/MTV-690[(MTV-690)]
+
+..MTV-759
+
+..MTV-848
+
+..MTV-624
+
+..MTV-681
+
+..MTV-680
+
+..MTV-689
+
+..MTV-733
+
+..MTV-669
+
+..MTV-715
+
+..MTV-714
+
+..MTV-696
+
+[id="known-issues-26_{context}"]
+== Known issues
+
+This release has the following known issues:
+
+.Deleting migration plan does not remove temporary resources
+
+Deleting a migration plan does not remove temporary resources such as importer pods, conversion pods, config maps, secrets, failed VMs, and data volumes. You must archive a migration plan before deleting it to clean up the temporary resources. link:https://bugzilla.redhat.com/show_bug.cgi?id=2018974[(BZ#2018974)]
+
+.Unclear error status message for VM with no operating system
+
+The error status message for a VM with no operating system on the *Plans* page of the web console does not describe the reason for the failure. link:https://bugzilla.redhat.com/show_bug.cgi?id=2008846[(BZ#22008846)]
+
+.Migration of virtual machines with encrypted partitions fails during conversion
+
+vSphere only: Migrations from {rhv-short} and OpenStack do not fail, but the encryption key might be missing on the target {ocp} cluster.
+
+
+.Migration fails during precopy/cutover while a snapshot operation is performed on the source VM
+
+Warm migrations from {rhv-short} fail if a snapshot operation is performed on the source VM. If a user performs a snapshot operation on the source VM at the time when a migration snapshot is scheduled, the migration fails instead of waiting for the user’s snapshot operation to finish. link:https://issues.redhat.com/browse/MTV-456[(MTV-456)]
+
+.Unable to schedule migrated VM with multiple disks to more than one storage classes of type hostPath
+
+When migrating a VM with multiple disks to more than one storage classes of type `hostPath`, a VM might not be scheduled. Workaround: Use shared storage on the target {ocp} cluster.
+
+.Non-supported guest operating systems in warm migrations
+
+Warm migrations and migrations to remote {ocp} clusters from vSphere do not support the same guest operating systems that are supported in cold migrations and migrations to the local {ocp} cluster. RHEL 8 and RHEL 9 might cause this limitation. +
+See link:https://access.redhat.com/articles/1351473[Converting virtual machines from other hypervisors to KVM with virt-v2v in RHEL 7, RHEL 8, and RHEL 9] for the list of supported guest operating systems.
+
+.VMs from vSphere with RHEL 9 guest operating system might start with network interfaces that are down
+
+When migrating VMs that are installed with RHEL 9 as a guest operating system from vSphere, the network interfaces of the VMs could be disabled when they start in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-491[(MTV-491)]
```

Comment:
This is badly written
```suggestion
When migrating VMs with RHEL 9 installed as a guest operating system from vSphere, the network interfaces of the VMs could be disabled when they start in {ocp-name} Virtualization. link:https://issues.redhat.com/browse/MTV-491[(MTV-491)]
```

Reply:
Also, we should define VMs like "virtual machines (VMs)" like in all the previous cases. Also, there's one fix, so there's my full suggestion:

"When migrating virtual machines (VMs) with Red Hat Enterprise Linux (RHEL) 9 installed as a guest operating system from vSphere, the network interfaces of these VMs could be disabled when they start in {ocp-name} Virtualization."